### DISCUSSION TITLE
Jandy Variable Speed Pumps

### DISCUSSION THREAD

**Discussion by Exit2Studios** [tagyoureit/nodejs-poolController#699] [Nixie]
In a strange situation, I've ended up replacing my blown Intelliflo power end with a Jandy ePump, which is controlled via a RS485 connection to a wi-fi controller.  I'm curious if anyone has any suggestions how to best incorporate this (if possible) into my njsPC setup.

**Comment by rstrouse** [comment]
Some of these pumps can be controlled by aqualinkd then through an interface.

**Reply by RRoyceus** [reply]
How do you interface AqualinkD?

**Reply by tagyoureit** [reply]
You follow the instructions for hooking up any EasyTouch to njsPC except use the RS485 on the AqualinkD.  njsPC will detect the Aqualink interface.

**Comment by Grauwulf75** [comment]
As far as I've been able to find out through my research, there is no way to control Jandy pumps via the RS485 connection at this point. You need to have one of Jandy's automation systems and use the AqualinkD project. Supposedly a Jandy SWG can be controlled via RS485 with NjsPC, but I haven't been able to get mine working at all.

**Comment by RRoyceus** [comment]
Please forgive me, but I have searched and cannot find the instructions for the EasyTouch.

Best Regards,

Jimmy Scruggs

The Scruggs Company
8530 Hansen Rd
Houston, TX 77075

Cell : 713-899-1862
Office: 713-649-2776
***@***.***

**Reply by RRoyceus** [reply]
I think we are having a failure to communicate.  I have the Nodejs Pool Control  hooked up and it works to control my Pentair VS pump.  Please see below.  In the interfaces tab there appears to be an option to connect to AquaLinkD.  I used the connection type as http, my IP address and port 80.  I assumed that I could use Nodejs Pool Control to connect to my jandy panel control my valves, chloinator, heater, etc as I do with AquaLinkD. 

![image](https://github.com/tagyoureit/nodejs-poolController/assets/33940987/5e3f055e-568c-4c71-ae68-ae58886c85f0)

**Reply by RRoyceus** [reply]
The IP address is the address of the the AquaLinkD

**Reply by tagyoureit** [reply]
The interfaces tab has an MQTT connectivity to AquaLinkD.  The directions I posted above are for the RS485 connection.

**Comment by tagyoureit** [comment]
All the instructions are in the [readme](https://github.com/tagyoureit/nodejs-poolController) and the [wiki](https://github.com/tagyoureit/nodejs-poolController/wiki).  The instructions are generic for any controller -- all controllers must be connected to an RS485 port on your computer and then njsPC will be able to communicate with it.
